diff --git a/node_modules/express/lib/application.js b/node_modules/express/lib/application.js
index cf6d78c..f9e177a 100644
--- a/node_modules/express/lib/application.js
+++ b/node_modules/express/lib/application.js
@@ -16,12 +16,12 @@
 var finalhandler = require('finalhandler');
 var debug = require('debug')('express:application');
 var View = require('./view');
-var http = require('node:http');
+var http = require('http');
 var methods = require('./utils').methods;
 var compileETag = require('./utils').compileETag;
 var compileQueryParser = require('./utils').compileQueryParser;
 var compileTrust = require('./utils').compileTrust;
-var resolve = require('node:path').resolve;
+var resolve = require('path').resolve;
 var once = require('once')
 var Router = require('router');
 
diff --git a/node_modules/express/lib/express.js b/node_modules/express/lib/express.js
index 2d502eb..b4ef299 100644
--- a/node_modules/express/lib/express.js
+++ b/node_modules/express/lib/express.js
@@ -13,7 +13,7 @@
  */
 
 var bodyParser = require('body-parser')
-var EventEmitter = require('node:events').EventEmitter;
+var EventEmitter = require('events').EventEmitter;
 var mixin = require('merge-descriptors');
 var proto = require('./application');
 var Router = require('router');
diff --git a/node_modules/express/lib/request.js b/node_modules/express/lib/request.js
index d8e5263..372a991 100644
--- a/node_modules/express/lib/request.js
+++ b/node_modules/express/lib/request.js
@@ -14,9 +14,9 @@
  */
 
 var accepts = require('accepts');
-var isIP = require('node:net').isIP;
+var isIP = require('net').isIP;
 var typeis = require('type-is');
-var http = require('node:http');
+var http = require('http');
 var fresh = require('fresh');
 var parseRange = require('range-parser');
 var parse = require('parseurl');
diff --git a/node_modules/express/lib/response.js b/node_modules/express/lib/response.js
index 9362d0e..ef1de94 100644
--- a/node_modules/express/lib/response.js
+++ b/node_modules/express/lib/response.js
@@ -16,11 +16,11 @@ var contentDisposition = require('content-disposition');
 var createError = require('http-errors')
 var encodeUrl = require('encodeurl');
 var escapeHtml = require('escape-html');
-var http = require('node:http');
+var http = require('http');
 var onFinished = require('on-finished');
 var mime = require('mime-types')
-var path = require('node:path');
-var pathIsAbsolute = require('node:path').isAbsolute;
+var path = require('path');
+var pathIsAbsolute = require('path').isAbsolute;
 var statuses = require('statuses')
 var sign = require('cookie-signature').sign;
 var normalizeType = require('./utils').normalizeType;
diff --git a/node_modules/express/lib/utils.js b/node_modules/express/lib/utils.js
index d53c5a1..5e608a2 100644
--- a/node_modules/express/lib/utils.js
+++ b/node_modules/express/lib/utils.js
@@ -12,7 +12,7 @@
  * @api private
  */
 
-var { METHODS } = require('node:http');
+var { METHODS } = require('http');
 var contentType = require('content-type');
 var etag = require('etag');
 var mime = require('mime-types')
diff --git a/node_modules/express/lib/view.js b/node_modules/express/lib/view.js
index d66b4a2..6beffca 100644
--- a/node_modules/express/lib/view.js
+++ b/node_modules/express/lib/view.js
@@ -14,8 +14,8 @@
  */
 
 var debug = require('debug')('express:view');
-var path = require('node:path');
-var fs = require('node:fs');
+var path = require('path');
+var fs = require('fs');
 
 /**
  * Module variables.
